java/yaks-testing/src/test/java/dev/yaks/testing/YaksTest.java

```diff
@@ -0,0 +1,21 @@
+package dev.yaks.testing;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.runner.RunWith;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        strict = true,
+        glue = {
+                "com.consol.citrus.cucumber.step.runner.core",
```

Comment:
I think that glue at some point should be added dynamically, e.g. injected from the operator via env variable. This way we can run user-defined glue code.

@christophd are those packages added to some place in the jar, e.g. by some plugin? If I include a Jar, is there a way to detect that there is glue code inside and enable the corresponding package?

Reply:
yes, we need that glue to be dynamic. I think Cucumber provides some mechanism to add dynamic glue.

For auto locating the glue packages inside a .jar I am not aware of any official way of doing this. We could introduce our own Java annotation on Step definitions and lookup all packages this way.

I think we need separate issues for these two points to not loose track of this.

Reply:
Added #43 and #44